app/helpers/application_helper.rb

```diff
@@ -11,14 +11,6 @@ def week(with_weekend = false)
     end
   end
 
-  def single_col_xs(inside, cols = 12)
-    content_tag :div, class: 'row' do
-      content_tag :div, class: "col-xs-#{cols}" do
-        inside
-      end
-    end
-  end
-
   def simple_error_notice(f)
     single_col_xs f.error_notification if f.error_notification.present?
```

Comment:
Since single_col_xs function is removed, shouldn't also the calls to it get removed? (There is also a call to it at the application helper.)